doc/source/roadmap.rst

```diff
@@ -72,7 +72,7 @@ In addition to making use of NumPy protocols like ``__array_function__``, we can
 make use of these protocols in SciPy as well.  That will make it possible to
 (re)implement SciPy functions like, e.g., those in ``scipy.signal`` for Dask
 or GPU arrays (see
-`NEP 18 - use outside of NumPy <http://www.numpy.org/neps/nep-0018-array-function-protocol.html#use-outside-of-numpy>`__).
+`NEP 18 - use outside of NumPy <https://www.numpy.org/neps/nep-0018-array-function-protocol.html#use-outside-of-numpy>`__).
```

Comment:
Making this https is better, but is there a way to reach this with intersphinx?

Reply:
Not with NumPy's current objects.inv, unfortunately, but it could be added.

Reply:
Perhaps editing the existing `numpy` entry is the right way to go. If we can check that all the existing `numpy` intersphinx references get resolved to the equivalent page at numpy.org, it sounds like a good thing to do (to me).

Reply:
The mapping for `numpy` was changed to https://numpy.org/devdocs with cbb7a08eea8131d7b46b0b9b985d187ae238eba0, which doesn't help with the https://numpy.org/neps/ links here.